nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/autodiff/samediff/ops/SDMath.java

```diff
@@ -5500,7 +5500,7 @@ public SDVariable tanh(String name, SDVariable x) {
 
   /**
    * Matrix trace operation<br>
-   * For rank 2 matrices, the output is a scalar with the trace - i.e., sum of the main diagonal.<br>
+   * For rank 2 matrices, the output is a scalar vith the trace - i.e., sum of the main diagonal.<br>
```

Comment:
Reintroducing a typo that was previously fixed? 

Given that this is a generated namespace, I guess you'll need to fix the typo in the definition code.